scripts/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_model.py

```diff
@@ -4,12 +4,14 @@
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
-from mantid import AlgorithmManager, logger
+from mantid import AlgorithmManager, AnalysisDataService, logger
```

Comment:
can we use https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/ADSHandler/ADS_calls.py